AEFeinstein/Swadge-IDF-5.0
# Add latest rawdraw_sf.h to fix memory leaks

### Description



`rawdraw_sf.h` was updated to the latest version. This fixes 2 memory leaks when running under X.

### Test Instructions



Run the emulator, check that it starts up OK, and especially on Linux check that it exits without asan reporting any leaks.

### Ticket Links



### Readiness Checklist
